### DISCUSSION TITLE
how to do something(like autocmd) after colorscheme switched by `space th`

### DISCUSSION THREAD

**Discussion by YuFei29** [NvChad/NvChad#2296] [Q&A] [answered]
Hello, everyone!
I want to change cursor color when I in neovim, so I put this code into `custom/init.lua`
``` lua
vim.api.nvim_create_autocmd({ "VimEnter", "VimResume", "ColorScheme" }, {
  pattern = { "*" },
  callback = function()
    vim.api.nvim_set_hl(0, "Cursor", { link = "St_NormalMode" })
    vim.api.nvim_set_hl(0, "CursorInsert", { link = "St_InsertMode" })
    vim.o.guicursor = "n-v-c-sm:block-Cursor,i-ci-ve:ver25-CursorInsert,r-cr-o:hor20"
  end,
})

vim.api.nvim_create_autocmd({ "VimLeave", "VimSuspend" }, {
  pattern = { "*" },
  callback = function()
    vim.api.nvim_set_hl(0, "Cursor", {})
    vim.o.guicursor = "a:block-Cursor"
  end,
})

```
it works very well except I can't change cursor color under normal mode when I use NvChad's buildin `Space-t-h` to switch colorscheme. 

Is there a way can let me change the cursor color after switch colorscheme? Or there other ways can do this? Thanks!

**Comment by siduck** [comment]
@YuFei29 why even using autocmds for this? try hl_override . read https://nvchad.com/docs/config/theming

**Reply by YuFei29** [reply, marked as answer]
It works! Thank you!
